x-pack/plugins/infra/server/lib/log_analysis/queries/common.ts

```diff
@@ -26,12 +32,10 @@ export const createTimeRangeFilters = (startTime: number, endTime: number) => [
   },
 ];
 
-export const createResultTypeFilters = (resultType: 'model_plot' | 'record') => [
+export const createResultTypeFilters = (resultTypes: Array<'model_plot' | 'record'>) => [
```

Comment:
Just curious: what's the difference/advantage here?

Reply:
I wanted to also use this helper in some places where we're filtering for more than one result type. Therefore the helper had to be extended to allow for that.